pkg/labels/selector.go

```diff
@@ -77,9 +62,9 @@ func (a ByKey) Less(i, j int) bool { return a[i].key < a[j].key }
 // Requirement implements both set based match and exact match
 // Requirement is initialized via NewRequirement constructor for creating a valid Requirement.
 type Requirement struct {
-	key       string
-	operator  Operator
-	strValues util.StringSet
+	Key       string
+	Operator  Operator
+	StrValues util.StringSet
```

Comment:
It's ok for it to stay as it is for api/types.go, but we'll need to consider the impact on the conversion functions.